modules/sitemaps/sitemap-builder.php

```diff
@@ -1253,14 +1253,14 @@ private function image_post_to_sitemap_item( $post ) {
 			),
 		);
 
-		$title = esc_html( $post->post_title );
+		$title = apply_filters( 'the_title_rss', $post->post_title );
```

Comment:
Could you add something like `/** This filter is already documented in wp-includes/feed.php */` to avoid tripping the parser?